mod.ts

```diff
@@ -131,6 +131,16 @@ export interface BuildOptions {
      * @default true
      */
     skipLibCheck?: boolean;
+    /**
+     * Deno enabled `isolatedModules` by default. but you can config `isolatedModules` for your js output
+     * @default true
+     */
+    isolatedModules?: boolean;
+    /**
+     * you can use tc39 [decorators](https://github.com/tc39/proposal-decorators) or typescript only `experimentalDecorators`
```

Comment:
I don't believe setting this to `false` will give you tc39 decorators (I think, or am I mistaken?). It's still to be seen how this is going to work and so I'd rather not expose it at this time.

Reply:
> I don't believe setting this to `false` will give you tc39 decorators (I think, or am I mistaken?). It's still to be seen how this is going to work and so I'd rather not expose it at this time.

you are right.
I open this options. just because maybe if some one want to use tc39 decorators in the future, the compileOptions maybe: `target: "esnext", experimentalDecorators: false`.